lib/runtime.js

```diff
@@ -62,6 +59,26 @@ export default class Runtime {
     const executionOptions = !options ? this.scriptOptions : options;
     const commandContext = CommandContext.build(this, executionOptions, codeContext);
 
+
+    // Will cooperate with Terminus to allow for inputs, if user has installed.
+    try {
+      var terminus = require('../../terminus/lib/terminus.js').provideTerminus();
```

Comment:
we should consume the [`terminusTerminal`](https://github.com/bus-stop/terminus/blob/a148e1d6211cdaa36f7bc06ecc5f9bb68545c200/package.json#L60) to get the terminal.

Reply:
for that matter there should be a standard `terminal` service that all terminals can provide to allow running in any terminal package.

Reply:
I have started creating PRs for terminal packages to provide the `terminal` service.

https://github.com/bus-stop/x-terminal/pull/236
https://github.com/bus-stop/terminus/pull/177
more to come...

Reply:
Could you make a PR fixing this?

Reply:
Is this still in works? It doesn't seem to be implemented in the actual release